<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartConfigGeneralWidget</class>
 <widget class="QWidget" name="PartConfigGeneralWidget">
  <property name="windowTitle">
   <string>Configure KtikZ Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="latexLabel">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the LaTeX executable here.&lt;/para&gt;</string>
        </property>
        <property name="text">
         <string>PDF&amp;LaTeX command:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>latexUrlRequester</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="latexUrlRequester">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the LaTeX executable here.&lt;/para&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pdftopsLabel">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the pdftops executable here.&lt;/para&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Pdftops command:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>pdftopsUrlRequester</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="pdftopsUrlRequester">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the pdftops executable here.&lt;/para&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Templates</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="replaceLabel">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the text which will be replaced by the TikZ code in the template here.&lt;/para&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Replace text:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>replaceEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="replaceEdit">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the text which will be replaced by the TikZ code in the template here.&lt;/para&gt;</string>
        </property>
        <property name="clearButtonEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="editorLabel">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the text editor for the template here.&lt;/para&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Editor command:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>editorUrlRequester</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="editorUrlRequester">
        <property name="whatsThis">
         <string>&lt;para&gt;Enter the path to the text editor for the template here.&lt;/para&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
